src/Explorer/Controls/Settings/SettingsSubComponents/__snapshots__/ScaleComponent.test.tsx.snap

```diff
@@ -39,13 +39,13 @@ exports[`ScaleComponent renders with correct intiial notification 1`] = `
     }
   >
     <ThroughputInputAutoPilotV3Component
-      canExceedMaximumValue={false}
+      canExceedMaximumValue={true}
       getThroughputWarningMessage={[Function]}
```

Comment:
Just pointing out that there's a functional change here and line 48 (the description doesn't mention it) and making sure it's intended.

Reply:
Yes. The change is the tests were running with dataExplorerPlatform === undefined but now we are pulling from configContext.platform which defaults to Portal